graspy/subgraph/subgraph.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
```

Comment:
@caseypw main comment - look at how the rest of the code in GraSPy is formatted into classes. All of your code should be in classes, with algorithm parameters passed to the __init__ of the class, and a .fit method like in sklearn. 

have a look here if you need more explanation https://scikit-learn.org/stable/developers/develop.html#apis-of-scikit-learn-objects

Reply:
I tried to format this code like the [simulations folder](https://github.com/neurodata/graspy/tree/master/graspy/simulations) and the [simulations.py](https://github.com/neurodata/graspy/blob/master/graspy/simulations/simulations.py) file in GraSPy. Should I not aim to emulate this code and rather look at classes in GraSPy?

Reply:
yes, this is more like the models in graspy like `SBMEstimator` or `AdjacencySpectralEmbed`